js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -1634,9 +1638,19 @@ module.exports = {
         controllers: {
             editing: {
                 editCell: function(rowIndex, columnIndex) {
-                    var isCellEditing = this.callBase(rowIndex, columnIndex),
-                        keyboardNavigationController = this.getController("keyboardNavigation");
+                    var keyboardNavigationController = this.getController("keyboardNavigation");
+
+                    if(keyboardNavigationController._canceledFocusCellPosition) {
```

Comment:
It is possible to make this code more compact:
```
var canceledFocusCellPosition = keyboardNavigationController._canceledFocusCellPosition;

if(canceledFocusCellPosition) {
    keyboardNavigationController._canceledFocusCellPosition = undefined;
    if(canceledFocusCellPosition.rowIndex === rowIndex && canceledFocusCellPosition.columnIndex === columnIndex) {
        return false;
    }
}
```

Reply:
I suppose that a separate method for this check should help